e2e/tcproute_test.go

```diff
@@ -727,3 +727,69 @@ func TestMeshWithMultipleTCPRoutes(t *testing.T) {
 		t.Fatalf("theCloud.TcpRoutes().Get(_, %v) = %v, nil, want err", removedTcpr.Key, tcpr)
 	}
 }
+
+func TestEnsureTcpRoute(t *testing.T) {
```

Comment:
`Ensure` test should have 2 parts:
1. create resources using rGraph
2. call planning on the same rGraph and check that no actions are needed (this way we ensure that GCE state equals desired state)

Reply:
Please take another look.